Python/Setup/swix/Packages/Microsoft.PythonTools.Profiling.Vsix.swixproj

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why did you rename it to *.vsix.swixproj? I recall running into a strange issue where I was forced to append "vsix", but it screwed up other things down the line (failed to copy the vsix properly). I managed to get around it by moving the MicroBuild.Core targets file declaration towards the end.